share/nisar/schemas/insar.yaml

```
@@ -76,6 +76,9 @@ runconfig:
             product_type: enum('GUNW', 'RIFG', 'RUNW', 'RIFG_RUNW_GUNW', 'ROFF_RIFG_RUNW_GOFF_GUNW', 'ROFF', 'GOFF', 'ROFF_GOFF')
             product_version: include('product_version_options', required=False)
             product_doi: include('product_doi_options', required=False)
+            # Data access for the static layers product associated with the output
+            # product (URL or DOI)
```

We updated the GSLC/GCOV runconfig docstring in #100. Could you please update here to be similar?

```suggestion
            # Data access for the static layers product associated with the
            # output product (URL or DOI). If not provided, the corresponding
            # metadata field will be set to "(NOT SPECIFIED)".
            # If this string contains the substring "{granule_id}", that substring
            # will be replaced with the granule ID.
```

https://github.com/isce-framework/isce3/blob/fecf943053d18eddcac9ded8e68a5f2f767b9673/share/nisar/defaults/gcov.yaml#L137C1-L141C52

---

Fixed. Thank you